features/blocked-chats/use-blocked-chats.ts

```diff
@@ -0,0 +1,40 @@
+import { getConversationDataQueryOptions } from "@/queries/use-conversation-data-query";
+import { useConversationListQuery } from "@/queries/useConversationListQuery";
+import { useCurrentAccount } from "@data/store/accountsStore";
+import { useQueries } from "@tanstack/react-query";
+import { useMemo } from "react";
+
+export const useBlockedChats = () => {
```

Comment:
I like what we discussed about the `ForCurrentAccount` suffix. Do you still like that?